missions/2PzD_Template_v3_0.VR/customization/geardefs/US/gearDefUSMC.sqf

```diff
@@ -39,10 +39,10 @@
 #define USMC_UniB_r             ["fow_u_usmc_p41_01_private"],["fow_u_usmc_p42_01_camo02_1_private"],["fow_u_usmc_p42_01_camo02_1_private"],["fow_u_usmc_p42_01_camo02_2_private"],["fow_u_usmc_p42_01_camo02_3_private"]
 
 //Backpack
-#define USMC_BP_r               ["fow_b_usmc_m1928"],["fow_b_usmc_m1928"],["fow_b_usmc_m1928"],["fow_b_usmc_m1928"],["B_LIB_US_M36"]
-#define USMC_BP_M1928           "fow_b_usmc_m1928"
+#define USMC_BP_r               "B_LIB_US_Backpack"
```

Comment:
Could you remove the "_r" on this one and subsequently change the add random chance call in the loadout to just be a regular add item call? Just to avoid confusion in the future since the "_r" is used elsewhere to indicate a list of items to randomly pick from.